pupil_src/launchables/eye.py

```diff
@@ -202,8 +202,30 @@ def get_timestamp():
         g_pool.get_timestamp = get_timestamp
         g_pool.get_now = get_time_monotonic
 
+        def load_runtime_pupil_detection_plugins():
+            from plugin import import_runtime_plugins
+            from pupil_detector_plugins.detector_base_plugin import PupilDetectorPlugin
+
+            plugins_path = os.path.join(g_pool.user_dir, "plugins")
+
+            for plugin in import_runtime_plugins(plugins_path):
+                if not isinstance(plugin, type(PupilDetectorPlugin)):
```

Comment:
The result of `type(PupilDetectorPlugin)` is `<class 'type'>`. I think, if you wanted to test for actual classes, this change might make your intent more clear:
```suggestion
                if not isinstance(plugin, type):
```